pgx-pg-sys/src/submodules/guard.rs

```diff
@@ -67,24 +66,33 @@ fn take_panic_location() -> PanicLocation {
 // via pg_module_magic!() this gets set to Some(()) for the "main" thread, and remains at None
 // for all other threads.
 #[cfg(debug_assertions)]
-thread_local! { pub(crate) static IS_MAIN_THREAD: once_cell::sync::OnceCell<()> = once_cell::sync::OnceCell::new() }
+thread_local! { pub(crate) static MAIN_THREAD: once_cell::sync::OnceCell<()> = once_cell::sync::OnceCell::new() }
+
+/// Return true when called from the same thread that executed [pg_module_magic!()]
```

Comment:
```suggestion
/// Return true when called from the same thread that executed [`pg_module_magic!()`]
```

Reply:
The square brackets create a link in the rendered doc.

Reply:
Although this doesn't actually work - `pgx::pg_module_magic` isn't in local scope